addons/l10n_sa_edi/tests/test_edi_zatca.py

```diff
@@ -1,17 +1,17 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
+from .common import TestSaEdiCommon
+
 from datetime import datetime
 from freezegun import freeze_time
-import logging
+from lxml import etree
 from pytz import timezone
 
 from odoo import Command
 from odoo.exceptions import ValidationError
 from odoo.tests import tagged
 from odoo.tools import misc
 
-from .common import TestSaEdiCommon
-
-_logger = logging.getLogger(__name__)
+import base64
```

Comment:
order :eyes: https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html#imports

I would do something like this:
```suggestion
import base64

from datetime import datetime
from freezegun import freeze_time
from lxml import etree
from pytz import timezone

from odoo import Command
from odoo.exceptions import ValidationError
from odoo.tests import tagged
from odoo.tools import misc
from odoo.addons.l10n_sa_edi.tests.common import TestSaEdiCommon
```